extensions/2.0/Vendor/EXT_structural_metadata/README.md

```diff
@@ -11,40 +13,522 @@
+>   }
+> }
+> ```
+> _Primitive_
+>
+> ```jsonc
+> {
+>   "primitives": [
+>     {
+>       "mode:": 0, 
+>       "attributes": {
+>         "POSITION": 0,
+>         "_DIRECTION": 1,
+>         "_MAGNITUDE": 2,
+>       },
+>       "extensions": {
+>         "EXT_structural_metadata": {
+>           "propertyMappings": [0]
+>         }
+>       }
+>     }
+>   ]
+> }
+> ```
+
+
+### Property Textures
+
+*Defined in [propertyTexture.schema.json](./schema/propertyTexture.schema.json).*
+
+Property textures use texture channels to store property values conforming to a particular class (identified by ID `class`), with those values accessed directly by texture coordinates. Property textures are especially useful when texture mapping high frequency data to less detailed 3D surfaces. Unlike textures used in glTF materials, property textures are not necessarily visible in a rendered scene. Like property tables, property textures are implementations of the [3D Metadata Specification](https://github.com/CesiumGS/3d-tiles/tree/main/specification/Metadata).
+
+Property textures are defined as entries in the `propertyTextures` array of the root-level `EXT_structural_metadata` extension, and may be referenced by extensions on primitive objects. 
+
+A property texture may provide channels for only a subset of the properties of its class, but class properties marked `required: true` must not be omitted.
+
+Several constraints are imposed to ensure compatibility of texture storage with various property types:
+
+* Variable-length arrays are not supported.
+* Data type and bit depth of the image must be compatible with the property type.
+
+Enum values may be encoded in images, as integer values according to their enum value type (see [Enum](#enum)).
+
+> **Implementation note:** Use of floating-point properties in a property texture would require a floating point-compatible image format like KTX2, provided by an additional extension.
+
+> **Example:** Property texture implementing a "wall" class, with property values stored in a glTF texture at index 0 and indexed by `TEXCOORD_0`. Each property of the `"wall"` class is stored in one channel of the texture.
+>
+> ![Property Texture](figures/property-texture.png)
+>
+> _Class and property texture_
+>
+> ```jsonc
+> {
+>   "extensions": {
+>     "EXT_structural_metadata": {
+>       "schema": {
+>         "classes": {
+>           "wall": {
+>             "name": "Wall Temperature vs. Insulation",
+>             "properties": {
+>               "insideTemperature": {
+>                 "name": "Inside Temperature",
+>                 "componentType": "UINT8"
+>               },
+>               "outsideTemperature": {
+>                 "name": "Outside Temperature",
+>                 "componentType": "UINT8"
+>               },
+>               "insulation": {
+>                 "name": "Insulation Thickness",
+>                 "componentType": "UINT8",
+>                 "normalized": true
+>               }
```

Comment:
Add `"type": "SCALAR"` to each property definition. Looks like I missed this in my first review.